tools/mapmerge2/merge_driver.py

```diff
@@ -11,6 +10,17 @@
 SELECT_RIGHT = 'right'
 
 
+def make_conflict_marker(type="/obj", name=""):
+    # Note that if you do not have an object that matches this path in
+    # your DME, the invalid path may be discarded when the map is loaded
+    # into a map editor. To rectify this, either add an object with this
+    # same path, or create a new object/denote an existing object in the
+    # obj_path define.
+    obj_path = f"{type}/merge_conflict_marker"
+    obj_name = f"Merge Conflict Marker{': ' + name if name else ''}"
```

Comment:
I'm not entirely sure about how tightly the `if` binds here
```suggestion
    obj_name = f"Merge Conflict Marker{(': ' + name) if name else ''}"
```

Reply:
Done